docs/REST/authentication/hmac.md

```diff
@@ -217,6 +217,52 @@ function signRequest(host,
 }
 ```
 
+### React native
+*Prerequisites*: [RNSimpleCrypto](https://github.com/ghbutton/react-native-simple-crypto), [base64](https://github.com/eranbo/react-native-base64)
```

Comment:
Thanks for your contribution! Could we use `crypto-js` library instead of `RNSimpleCrypto`? The former is a well-known library with 4000+ public usages as compared to the latter that does not seem to have any public usage.

Link : https://www.npmjs.com/package/crypto-js

Reply:
@abhilasharora `crypto-js` library is not compatible with `react-native` which is why I had to use another library in place.
The port of `crypto-js` for react native is [`react-native-crypto-js`](https://github.com/imchintan/react-native-crypto-js), but it doesn't supports hmac256 (see it's readme)
Other libs available are:
- [react-native-crypto](https://github.com/tradle/react-native-crypto) the reason for not using this package is that it has a lot of dependency installations before you can use. Refer to installation in readme. Another valid reason is the list of issues filed on github
- [react-native-fast-crypto](https://github.com/EdgeApp/react-native-fast-crypto) doesn't supports hmac - [see issues](https://github.com/EdgeApp/react-native-fast-crypto/issues/14)
- [react-native-crypto](https://github.com/trackforce/react-native-crypto) has issues with pod install - [see issues](https://github.com/trackforce/react-native-crypto/issues/14)
- [react-native-crypto](https://github.com/ghbutton/react-native-simple-crypto) this is a fork of [react-native-crypto](https://github.com/trackforce/react-native-crypto) but it's updated and works great without issues.

As for `base64`, since `crypto-js` was incompatible I had to use another [library](https://github.com/eranbo/react-native-base64) available in place which is pretty standard.